include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -388,7 +388,7 @@ class alphabet_tuple_base :
      * with one of its derived classes (seqan3::qualified).
      * \include test/snippet/alphabet/composite/alphabet_tuple_base_value_assignment.cpp
      */
-    template <typename component_type>
+    template <typename component_type alph>
```

Comment:
this is syntactically wrong. 

you try to fix this warning:
```
.../seqan3/include/seqan3/alphabet/composite/alphabet_tuple_base.hpp:412: warning: argument 'alph' of command @param is not found in the argument list of seqan3::structured_aa< sequence_alphabet_t, structure_alphabet_t >::operator=(typename indirect_component_type)
```
but I think this warning warns you about a different location, namely the `operator=` of `seqan3::structured_aa`, which is apparently in https://github.com/seqan/seqan3/blob/ddda01f410b8dc83e664ab67a3ca6a87bd28d584/include/seqan3/alphabet/structure/structured_aa.hpp#L81-L88

My guess is that we need to add the templates and that a block comment would be appropiate:

```c++
#if SEQAN3_DOXYGEN_ONLY(1)0 // this complete block is only visible by doxygen.
    //!\copydoc alphabet_tuple_base::alphabet_tuple_base(component_type const alph)
    template <typename component_type>
    constexpr structured_aa(component_type const alph) {}

    //!\copydoc alphabet_tuple_base::alphabet_tuple_base(indirect_component_type const alph)
    template <typename indirect_component_type>
    constexpr structured_aa(indirect_component_type const alph) {}

    //!\copydoc alphabet_tuple_base::operator=(component_type const alph)
    template <typename component_type>
    constexpr structured_aa & operator=(component_type const alph) {}

    //!\copydoc alphabet_tuple_base::operator=(indirect_component_type const alph)
    template <typename indirect_component_type>
    constexpr structured_aa & operator=(indirect_component_type const alph) {}
#endif
```